interface/patient_file/summary/create_portallogin.php

```diff
@@ -169,11 +169,9 @@ function filterTwigTemplateData($twig, $pid, $templateName, $data)
 
     array_push($query_parameters, $pid);
     if (sqlNumRows($res)) {
-        // TODO: @adunsulag is there a reason why we don't audit the fact that the patient credentials were created and updated?
-        // That seems like a pretty important security event we want to be aware of.
```

Comment:
pretty sure i didn't log these into audit engine since this means the hashes will get stored in the log table and be able to view them via log reports. prob best not to do that. if need to audit changes would instead make a newEvent() call via EventAuditLogger to log this manually (then it won't bring in the sensitive info).